apps/backoffice-v2/src/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/useProcessTrackerLogic.tsx

```diff
@@ -1,127 +1,61 @@
-import { useCallback, useEffect, useMemo, useState } from 'react';
-import {
-  pluginsWhiteList,
-  processStatusToIcon,
-  tagToAccordionCardItem,
-  tagToIcon,
-} from '@/common/components/molecules/ProcessTracker/constants';
-import { titleCase } from 'string-ts';
-import { TWorkflowById } from '@/domains/workflows/fetchers';
-import { valueOrNA } from '@/common/utils/value-or-na/value-or-na';
-import { ProcessStatus } from '@ballerine/common';
+import { tagToAccordionCardItem } from '@/common/components/molecules/ProcessTracker/constants';
 import { IUseProcessTrackerLogicParams } from '@/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/interfaces';
+import { processTrackersMap } from '@/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters';
+import { IProcessTracker } from '@/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters/process-tracker.abstract';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+const defaultProcesses = ['collection-flow', 'third-party', 'ubos'];
```

Comment:
We are better off without a default. These options seem too specific to be a default.

Reply:
P1